force-app/main/default/lwc/stoRegisterThread/stoRegisterThread.js

```diff
@@ -650,7 +650,7 @@ export default class StoRegisterThread extends NavigationMixin(LightningElement)
         if (!this.openThreadList) return '';
         return this.canOpenMoreThreads
             ? `Vi ser at du allerede har ${this.openThreads} åpne meldinger på dette temaet. Ønsker du å fortsette på en tidligere melding eller ønsker du å skrive en ny?`
-            : `Vi ser at du allerede har ${this.openThreads} åpne meldinger på dette temaet ${this.capitalizedCategory}. Du kan maks ha 3 samtaler på hvert tema. Hvis du vil opprette en ny samtale, må du derfor avslutte en av de du allerede har.`;
+            : `Vi ser at du allerede har ${this.openThreads} åpne meldinger på dette temaet ${this.capitalizedCategory}. Du kan maks ha tre samtaler på hvert tema. Hvis du vil opprette en ny samtale, må du derfor avslutte en av de du allerede har.`;
```

Comment:
Bare meg som synes den kommaen er rar? Ser vi hadde det før og. Tror dette er mer riktig, men må vel gå via kommunikasjon-
Hvis du vil opprette en ny samtale, må du derfor avslutte en av de du allerede har.
```suggestion
            : `Vi ser at du allerede har ${this.openThreads} åpne meldinger på dette temaet ${this.capitalizedCategory}. Du kan maks ha tre samtaler på hvert tema. Hvis du vil opprette en ny samtale må du derfor avslutte en av de du allerede har.`;
```

Reply:
Enig i at den er rar. Jeg høre med Lena.